libs/perun/components/src/lib/facilities-list/facilities-list.component.ts

```diff
@@ -41,11 +41,21 @@ export class FacilitiesListComponent implements OnChanges {
 
   dataSource: MatTableDataSource<EnrichedFacility>;
   localDisableRouting: boolean;
+  _facilityWithAuthzGroupPairs = new Map<number, Group[]>();
+  _authzVoNames: Map<number, string>;
 
   private sort: MatSort;
 
   constructor(private authResolver: GuiAuthResolver) {}
 
+  @Input() set facilityWithAuthzGroupPairs(pairs: Map<number, Group[]>) {
+    this._facilityWithAuthzGroupPairs = pairs;
+  }
+
+  @Input() set authzVoNames(authzVoNames: Map<number, string>) {
+    this._authzVoNames = authzVoNames;
+  }
```

Comment:
It is not technically wrong to use input setter like this, it is more useful when there is a peace of code that needs to run every time the input changes. However to stay consistent with the rest of the codebase could you please make these a simple `@Input`. This goes for other list components.